server/src/shared/Entities/tags.entity.ts

```diff
@@ -0,0 +1,34 @@
+// Other dependencies
+import {
+    Column,
+    Entity,
+    ObjectID,
+    ObjectIdColumn,
+    CreateDateColumn,
+    UpdateDateColumn,
+} from 'typeorm'
+
+@Entity('Tags')
+export class TagsEntity {
+    constructor(partial: Partial<TagsEntity>) {
+        Object.assign(this, partial)
+    }
+
+    @ObjectIdColumn()
+    _id: ObjectID
+
+    @Column({ type: 'string' })
```

Comment:
We should make the name as unique, like:
```suggestion
    @Column({ type: 'string', unique: true })
```